### PR TITLE
Task #150780 feat: Add sendemail config colum level.

### PR DESCRIPTION
**Pull request by komalm**

### CODE REVIEW

File: tjreports/site/assets/js/tjrContentUI.js [hunk 1]
```diff
@@ -45,6 +46,14 @@ jQuery.extend(tjrContentUI.report, {
 				var responseHTML = jQuery(response['html']).find(containerSel).html();
 				jQuery(containerSel).html(responseHTML);
 
+				// If sendEmail plug is enable then call the below funcation. To show checkboxes
+				var isSendEmail = jQuery('body').find('.td-sendemail').length;
+				if (isSendEmail)
```
Comment (coolbung) [inline tjreports/site/assets/js/tjrContentUI.js]: Also add a check for tjutilitysendemail defined to handle cases when the plugin is unpublished or not present.

File: tjreports/site/models/reports.php [hunk 2]
```diff
@@ -1065,15 +1077,15 @@ private function filterReportColumns($queryId, &$selColToshow)
 		}
 
 		$query = $this->_db->getQuery(true);
-		$showhideCols = $paramColToshow = array();
+		$showhideCols = $paramColToshow = $emailColumn = array();
```
Comment (coolbung) [inline tjreports/site/models/reports.php]: why is `$emailColumn` an array ?

File: tjreports/site/views/reports/tmpl/default.php [hunk 3]
```diff
@@ -431,7 +433,12 @@
 													}
 													else
 													{
-														echo '<td class="' . $key . '">'. $item[$key] .'</td>';
+														$isSendEmailClass = '';
```
Comment (coolbung) [inline tjreports/site/views/reports/tmpl/default.php]: Change this to 
`$isSendEmailClass = (in_array($key, $this->emailColumn)) ? $emailColmClass : '';`

and
`echo "<td class=\"{$key} {$isSendEmailClass}  \">{$item[$key]}</td>";`

File: tjreports/site/views/reports/tmpl/default.php [hunk 4]
```diff
@@ -431,7 +433,12 @@
...
+														if (in_array($key, $this->emailColumn))
+														{
+															$isSendEmailClass = $emailColmClass;
+														}
+														echo '<td class="' . $key . ' ' . $isSendEmailClass . '">'. $item[$key] .'</td>';
```
[the remark this answers is not in the document]
Comment (komalm) [reply tjreports/site/views/reports/tmpl/default.php]: It's already there I am just have added class.

**Pull request merged**